src/simtools/visualization/plot_mirrors.py

```diff
@@ -153,6 +150,7 @@ def plot_mirror_layout(mirrors, telescope_model_name, title=None):
 
     x_pos = mirrors.mirror_table["mirror_x"].to("cm").value
     y_pos = mirrors.mirror_table["mirror_y"].to("cm").value
+    x_pos, y_pos = y_pos, -x_pos
```

Comment:
Suggest to add a comment here to motivate the rotation. We otherwise won't remember.